app/controllers/items_controller.rb

```diff
@@ -220,7 +220,13 @@ def set_rights
   def apply_apo_defaults
     Dor::Services::Client.object(@cocina.externalIdentifier).apply_admin_policy_defaults
     reindex
-    render status: :ok, plain: 'APO defaults applied.'
+    respond_to do |format|
+      if params[:bulk]
+        format.html { render status: :ok, plain: 'APO defaults applied.' }
+      else
+        format.any { redirect_to solr_document_path(params[:id]), notice: 'APO defaults applied!' }
+      end
+    end
   rescue Dor::Services::Client::UnexpectedResponse => e
```

Comment:
This method is currently only used by the bulk action -- this lets us re-purpose it for the new button.